src/controllers/dlgprefcontrollers.cpp

```diff
@@ -19,13 +19,8 @@ DlgPrefControllers::DlgPrefControllers(DlgPreferences* pPreferences,
     setupUi(this);
     setupControllerWidgets();
 
-    connect(&m_buttonMapper, SIGNAL(mapped(QString)),
-            this, SLOT(slotOpenLocalFile(QString)));
-
-    connect(btnOpenUserPresets, SIGNAL(clicked()),
-            &m_buttonMapper, SLOT(map()));
-
-    m_buttonMapper.setMapping(btnOpenUserPresets, userPresetsPath(m_pConfig));
+    connect(btnOpenUserPresets, &QPushButton::clicked,
+            this, [this] { slotOpenLocalFile(userPresetsPath(m_pConfig)); });
```

Comment:
Here we can also copy the userPresetsPath into a local variable and capture it by value